### PR TITLE
Fixes #2455 - Improve onboarding docs for new contributors

### LINKED ISSUE #2455: Error setting up this project locally for development
# Steps to reproduce

I’m on MacOS

```
$ git clone git@github.com:webcompat/webcompat.com.git
$ cd webcompat.com.git
$ npm run setup
$ cp config/secrets.py.example config/secrets.py
$ npm start

[…]
Statuses Initialization…
Fetching milestones from Github…
It failed with 404 Client Error: Not Found for url: https://api.github.com/repos/milestones!
We will read from data/milestones.json.

Oooops.
We can't find /Users/mkuehnel/Documents/Projects/webcompat.com/data/milestones.json
Double check that everything is configured properly
in config/secrets.py and try again. Good luck!
[…]
```

# Reason

<https://api.github.com/repos/milestones> returns:

```
{
  "message": "Not Found",
  "documentation_url": "https://developer.github.com/v3"
}
```

# Potential fix

Guess it should be <https://api.github.com/repos/webcompat/webcompat.com/milestones>, right?

### PR DESCRIPTION
**Pull request by mischah**
I had some problems to get the server running to start contributing.

The changes of this PR might help others.

Closes #2455

### CODE REVIEW

File: docs/dev-env-setup.md [hunk 1]
```diff
@@ -185,7 +191,13 @@ or
 npm run start
 ```
 
-You should now have a local instance of the site running at `http://localhost:5000/`. Please [file bugs](https://github.com/webcompat/webcompat.com/issues/new) if something went wrong!
+You should now have a local instance of the site running at `http://localhost:5000/`.
+
+#### Getting error messages?
+
+First you should have a look at the logs. They are located at `/tmp/webcompat.log`.
```
Comment (zoepage) [inline docs/dev-env-setup.md]: or ping on IRC or slack https://webcompat.com/contact

Comment (mischah) [reply docs/dev-env-setup.md]: The log location is defined over here:

Comment (miketaylr) [reply docs/dev-env-setup.md]: this comment probably makes most sense after "Please file bugs"

**Comment by mischah** [comment]
> this comment probably makes most sense after "Please file bugs"

I flipped that two sentences with 1c9d0cf.

**Comment by karlcow** [comment]
@mischah I changed the title of the PR.
Could you just merge your two commit in one and change the commit message to be 

`Issue #2455 - Improves onboarding docs for new contributors`

Thanks.

In your local branch, you can do:

```
git reset HEAD~2
# change the second commit lines to start with a `f`
# change the first commit message to be the new `Issue #2455 - Improves onboarding docs for new contributors`
git push -f improve-dev-docs
```

Hope it helps.

**Comment by mischah** [comment]
> Could you just merge your two commit in one and change the commit message

Done 😘

**Comment by miketaylr** [comment]
thank you!

**Pull request merged**